### ISSUE TITLE
Focus is not visible in a high contrast mode

### ISSUE THREAD

**Issue by ekaterina-78** [telekom/scale#1125]
**Scale Version**
3.0.0-beta.40 (the problem remains in newer versions as well)

**Framework and version**
Vue

**Current Behavior**
Focus on checkboxes and buttons is not detectable in windows high contrast mode

**Expected Behavior**
Focus on checkboxes and buttons is easily noticeable

**Steps to reproduce**
Turn on High Contrast on Windows (Shift+Alt+PrtSc or set it in the settings)
Focus a checkbox or a button.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome - Version 103.0.5060.114 (Official Build) (64-bit)

**Additional context**
Hello,
it looks like scale checkboxes and buttons (close button on modal window as well) do not fully fullfill accessibility criteria.
When these elements get focus in a high contrast mode, it is not detectable (there is no difference between an element with and without focus).
![image](https://user-images.githubusercontent.com/84175855/177754383-b8781855-595a-4aa1-a911-df54dfcb484b.png)
![image](https://user-images.githubusercontent.com/84175855/177754417-d4bc6464-11f8-4c1c-a580-c12422f5d1d8.png)

Focus for checkboxes and buttons is implemented with the box-shadow, which is not visible in Windows high-contrast themes. Possible solution - additionally adding a transparent solid outline, when checkboxes and buttons get focus.

**Comment by Kaawo** [comment]
Greetings,

same issue. Are there any updates?

Kind regards
Kay

**Comment by felix-ico** [comment]
we've reworked the focus states for all components and this should now work as expected in HCM as well
